spec/support/shared/unit/provider/file.rb

```diff
@@ -466,10 +466,12 @@ def unlink
       }
 
       let(:verification) { double("Verification") }
+      let(:verification_fail) { double("Verification Fail") }
```

Comment:
I'd like to start using `instance_double` wherever possible. https://www.relishapp.com/rspec/rspec-mocks/v/3-3/docs/verifying-doubles/using-an-instance-double